### PR TITLE
epass2003: use true random number for mutual authentication for SM.

### PR DESCRIPTION
**Pull request by popovec**
modified:   src/libopensc/card-epass2003.c

### CODE REVIEW

File: src/libopensc/card-epass2003.c [hunk 1]
```diff
@@ -39,9 +39,10 @@
 #include <stdlib.h>
 #include <string.h>
 
+#include <openssl/cmac.h>
```
Comment (frankmorgner) [inline src/libopensc/card-epass2003.c:42]: this include does not seem to be needed

Comment (popovec) [reply src/libopensc/card-epass2003.c:42]: We need this #include <openssl/cmac.h> for function aes128_encrypt_cmac() - epass2003 uses CMAC-AES-128 for SM in FIPS certified mode. (I changed the order of includes in this file, as git-clang-format had a problem with another order.)

Comment (popovec) [reply src/libopensc/card-epass2003.c:42]: Ops, incorrect "include"... The PR is already fixed, modified, the unnecessary #include <openssl/evp.h>" is removed.

**Review by frankmorgner — APPROVED** [review]

**Review by Jakuje — APPROVED** [review]
Good catch!

**Pull request merged**